lib/models/angular-module.ts

```diff
@@ -1,12 +1,3 @@
-import { Type } from '@angular/core';
+import { ModuleWithProviders, Type } from '@angular/core';
 
-// Angular 5x has ModuleWithProviders and Angular 6 uses a generic ModuleWithProviders<T>.
```

Comment:
I noticed these old NG5 reference still lying around and removed them since we don't support NG5 in master anymore.